### DISCUSSION TITLE
2.3.52.iso trojan found

### DISCUSSION THREAD

**Discussion by mhackelton** [Security-Onion-Solutions/securityonion#4532] [General] [answered]
FYI: McAfee VSE found the Artemis!EE468A4B1F55 Trojan in default_exe.exe, in the 2.3.52.iso file downloaded from this site. Have downloaded twice, and verified the MD5 hash.

**Comment by dougburks** [comment, marked as answer]
That is a false positive.  Please see the Warning on the Download page of the documentation:
https://docs.securityonion.net/en/2.3/download.html

Also see:
https://github.com/Security-Onion-Solutions/securityonion/discussions/4306
https://github.com/Security-Onion-Solutions/securityonion/discussions/4235

**Reply by mhackelton** [reply]
Appreciate the info, thanks!
